src/jollyjumper/core.py

```diff
@@ -12,26 +11,40 @@ def get_enjambment(text):
     :param text: String with the original poem
     :return: Dictionary of all enjambment types and their lines of occurrence
     """
-    enjambment_types = ['sirrematic']
+    enjambment_types = ['sirrematic', 'sirrematic_relation_words_preposition',
+                        'sirrematic_relation_words_conjunction',
+                        'sirrematic_relation_words_determiners',
+                        'sirrematic_orational', 'relation_words_verbs',
+                        'sirrematic_with_verb', 'link',
+                        'sirrematic_prepositional_without_de',
+                        'sirrematic_prepositional',
+                        'sirrematic_prepositional_prep_before_noun_or_adjective']
     enjambments = {}
     nlp = load_pipeline()
     doc = nlp(text)
     for token in doc:
-        # We look for tmesis before any other enjambment type because the text has to be preprocessed
+        # We look for tmesis before any other enjambment type because the text
+        # has to be preprocessed
         if token._.has_tmesis:  # noqa
-            enjambments[token._.line] = ('tmesis', token.text.split('-\n'))  # noqa
+            enjambments[token._.line] = (
+            'tmesis', token.text.split('-\n'))  # noqa
             continue
-        # Last token cannot be the beginning of an enjambment (unless it's tmesis)
-        if token is doc[-1]:
+        # Last token cannot be the beginning of an enjambment
+        # (unless it's tmesis)
+        if token == doc[-1]:
             continue
-        previous_token = doc[token.i-1]
+        previous_token = doc[token.i - 1]
         next_token = doc[token.i + 1]
-        # We look for enjambment when if there are words before and after a newline character
-        if token.text == '\n' and not previous_token.is_punct and not next_token.is_punct:
+        # We look for enjambment when if there are words
+        # before and after a newline character
+        if token.text == '\n' \
+                and not previous_token.is_punct and not next_token.is_punct:
             for enjambment_type in enjambment_types:
```

Comment:
Let's try to avoid using `\` and instead use parenthesis and indentation.

Reply:
Done